cluster/schema/schema.go

```diff
@@ -742,8 +742,5 @@ func (s *schema) deleteAlias(alias string) error {
 }
 
 func (s *schema) unsafeResolveClass(class string) *metaClass {
-	if cls := s.unsafeResolveAlias(class); cls != "" {
```

Comment:
This is the main change. When doing tenants or property updates (which we want to avoid) we use use schema reader's ReadOnlyClass() api that automatically resolves alias. 

By removing this. Auto resolve won't happen hence collection manipulation can only be done via "collection" name not alias name.

I fixed few tests. But no other test fails.